client/settings/disable-upe-modal/index.js

```diff
@@ -88,28 +88,12 @@ const DisableUpeModalBody = ( { enabledMethods } ) => {
 	);
 };
 
-const DisableSubmitButton = () => {
```

Comment:
Since `DisableUpeModal` already contains the usage for `useIsUpeEnabled()` and `useContext( WcPayUpeContext )`, might as well inline it 🤷